packages/core/content-manager/server/src/history/services/utils.ts

```diff
@@ -8,6 +8,11 @@ import { HistoryVersions } from '../../../../shared/contracts';
 
 const DEFAULT_RETENTION_DAYS = 90;
 
+type RelationResponse = {
+  results: any[];
+  meta: { missingCount: number };
+};
```

Comment:
I think you can use [RelationResult](https://github.com/strapi/strapi/blob/301d2027a59183bfa76012748eaeef6b5ffae9d1/packages/core/content-manager/shared/contracts/relations.ts#L6) for relations. For media assets I think there should be a type somewhere too